source/Applications/Advanced/TransformViaArucoMarker/TransformViaArucoMarker.cpp

```diff
@@ -0,0 +1,315 @@
+/*
+This example shows how to transform a point cloud using the pose of an aruco marker.
+*/
+
+#include <Zivid/Experimental/Calibration.h>
+#include <Zivid/Visualization/Visualizer.h>
+#include <Zivid/Zivid.h>
+
+#include <algorithm>
+#include <opencv2/core/core.hpp>
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+
+#include <aruco_nano.h>
+#include <cmath>
+#include <iostream>
+
+namespace
+{
+    struct Line2d
+    {
+        double slope;
+        double intercept;
+    };
+
+    int clamp(const int n, const int lower, const int upper)
+    {
+        return std::max(lower, std::min(n, upper));
+    }
+
+    // Fit a line y=a*x + b to 2 points
```

Comment:
In Python samples function headers are inside the function definition, check https://github.com/zivid/zivid-python-samples/blob/6c085911df130417732389d973ec90cded731de7/source/applications/advanced/gamma_correction.py#L28-L50
Please check if this should be the case with C++ as well

Reply:
putted inside the function. This is what we do in other c++ samples.